app/build.gradle

```diff
@@ -68,7 +69,10 @@ dependencies {
     implementation 'com.google.firebase:firebase-core:17.0.0'
     implementation 'com.firebaseui:firebase-ui-auth:4.3.1'
     implementation 'androidx.recyclerview:recyclerview:1.0.0'
+    implementation 'com.buglife.sdk:buglife-android:1.5.1'
 
+    // Buglife dependencies
+    implementation 'com.buglife.sdk:buglife-android:1.5.1'
```

Comment:
why have it twice?

Reply:
it does not appear outdated here but i removed the extra dependency of buglife. Sorry about that